src/client.rs

```diff
@@ -15,6 +17,8 @@ pub struct Client {
 
     /// The key service handling key related serialization and deserialization.
```

Comment:
```suggestion
    /// The key service handling key related requests.
```
Can you get this one for me too? Just to keep them all consistent.